build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id 'java-library'
```

Comment:
Also jacoco, to use later

Reply:
likewise, we can add when we actually get some tests :)